app/views/taxonomy_signups/new.html.erb

```diff
@@ -0,0 +1,40 @@
+<% content_for :title, @taxon['title'] %>
+
+<%= render 'govuk_component/breadcrumbs', breadcrumbs: @breadcrumbs %>
+<div class='column-two-thirds'>
+
+<% if @taxon.dig('links', 'child_taxons').present?%>
+  <div class="govuk-title length-long">
+   <h1>What do you want to get alerts about?</h1>
+  </div>
+
+  <%= form_tag({action: "confirm"}, method: "get") do %>
+    <fieldset>
+      <legend id="alerts" class="visually-hidden">What do you want to get alerts about?</legend>
+      <div class='email-signup-taxons' role='radiogroup' aria-labelledby='alerts'>
+        <p class="email-option">Everything published to GOV.UK about:</p>
+
+        <%= render partial: 'taxon', locals: { taxon: @taxon, is_parent: true } %>
+
+        <p class='email-option form-block'>or only one of the following</p>
+
+        <% @taxon.dig('links', 'child_taxons').each_with_index do |taxon, taxon_index| %>
```

Comment:
might be nicer to use:

```
<%= render partial: "taxon", collection: @taxon.dig('links', 'child_taxons') %>
```
this would give youn the `index` variable in then view rather than having to pass it in.